### PR TITLE
Switch to katgpucbf image

### PR DESCRIPTION
**Pull request by bmerry**
- Rename xgpu to xbgpu everywhere
- No longer need to use capambel for fgpu, since it runs as root inside
  the container.
- Pass katcp port for fgpu.

Closes NGC-263.

### CODE REVIEW

File: katsdpcontroller/generator.py [hunk 1]
```diff
@@ -344,15 +343,15 @@ def _make_fgpu(
             '--src-affinity', '{cores[src0]},{cores[src1]}',
             '--dst-interface', '{interfaces[cbf].name}',
             '--dst-affinity', '{cores[dst]}',
-            # TODO: reenable once katxgpu can handle it
+            # TODO: reenable once katgpucbf.xbgpu can handle it
             # '--dst-packet-payload', '8192',
```
Comment (james-smith-za) [inline katsdpcontroller/generator.py:347]: Is there a JIRA ticket or some TODO note somewhere to remind us to do this?

File: katsdpcontroller/generator.py [hunk 2]
```diff
@@ -383,47 +382,47 @@ def _make_fgpu(
...
+        xbgpu.cores = ['core']
+        xbgpu.interfaces = [scheduler.InterfaceRequest('cbf', infiniband=ibv)]
+        xbgpu.interfaces[0].bandwidth_in = acv.data_rate() / n_engines
+        xbgpu.interfaces[0].bandwidth_out = stream.data_rate() / n_engines
+        xbgpu.gpus = [scheduler.GPURequest()]
+        xbgpu.gpus[0].compute = 0.125  # TODO: scale according to problem size
+        xbgpu.gpus[0].mem = 1024       # TODO: check what it really needs
         # Minimum capability as a function of bits-per-sample, based on
-        # tensor_core_correlation_kernel.mako from katxgpu.
+        # tensor_core_correlation_kernel.mako from katxbgpu.
```
Comment (james-smith-za) [inline katsdpcontroller/generator.py:412]:
```suggestion
        # tensor_core_correlation_kernel.mako from katgpucbf.xbgpu.
```

**Review by james-smith-za — APPROVED** [review]
One tiny-tiny bikeshed suggestion, which you may take or leave, up to you.

Otherwise looks good.

File: katsdpcontroller/generator.py (hunk 2 above)
Comment (bmerry) [reply katsdpcontroller/generator.py:412]: Thanks, I missed this one.

File: katsdpcontroller/generator.py (hunk 1 above)
Comment (bmerry) [reply katsdpcontroller/generator.py:347]: NGC-208.

**Pull request merged**